### PR TITLE
Release notes for Foreman 3.2.1

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

File: _includes/manuals/3.2/1.2_release_notes.md [hunk 1]
```diff
@@ -31,6 +31,37 @@ Now that Debian 11 is supported, Debian 10 support is deprecated and will be dro
 
 Note that this support statement refers to running Foreman and Foreman Proxy themselves on Debian 10. Managing Debian 10 hosts remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
 
+### Release notes for 3.2.1
+#### Foreman - Host registration
+* The registration database migration could fail when template is not available ([#34661](https://projects.theforeman.org/issues/34661))
```
Comment (melcorr) [inline _includes/manuals/3.2/1.2_release_notes.md]:
```suggestion
* The registration database migration could fail when the template is not available ([#34661](https://projects.theforeman.org/issues/34661))
```

File: _includes/manuals/3.2/1.2_release_notes.md [hunk 2]
```diff
@@ -31,6 +31,37 @@ Now that Debian 11 is supported, Debian 10 support is deprecated and will be dro
...
+
+#### Foreman - Reporting
+* Remove reports deprecation from develop and 3.2 ([#34744](https://projects.theforeman.org/issues/34744))
+* Prepare Foreman core for Host Reports migration ([#34360](https://projects.theforeman.org/issues/34360))
+
+#### Foreman - TFTP
+* Make the operatingsystem.bootfile usable in Jail ([#34689](https://projects.theforeman.org/issues/34689))
+
+#### Foreman - Web Interface
+* Breadcrumb switcher doesn't work with katello content ([#34495](https://projects.theforeman.org/issues/34495))
```
Comment (melcorr) [inline _includes/manuals/3.2/1.2_release_notes.md]:
```suggestion
* Breadcrumb switcher doesn't work with Katello content ([#34495](https://projects.theforeman.org/issues/34495))
```

**Review by melcorr — COMMENTED** [review]
Tiny nits that are not blockers in any way.

**Comment by upadhyeammit** [comment]
Corrected the titles and updated redmine issues accordingly, thanks @melcorr !

**Pull request merged**